projects/dsp-ui/src/assets/style/_viewer.scss

```diff
@@ -6,6 +6,11 @@
   background-color: $black-12-opacity;
 }
 
+.selected-resource {
+  background-color: $grey;
+  opacity: $black-12-opacity;
```

Comment:
it's not the opacity. `$black-12-opacity` is a color with already defined opacity of 0.12. It should be the same as on line 6 in this file. You can also combine with the '.link:hover' selector:

```scss
.link:hover,
.selected-resource {
  background-color: $black-12-opacity;
}

Reply:
I checked it before but then the background colour for hover and selected options look same. To distinguish the selected option I used grey colour with some opacity as there are not many colours defined in config file.

If you think it is fine to have same colour for both hover and selected option, I can make your suggested changes.

Reply:
Yes I know. Then you should use `$black-20-opacity` to make a difference to the hover color.

Reply:
Use `background-color: $black-20-opacity;` on line 10 and delete opacity property on line 11.

Reply:
$black-12-opacity and $black-20-opacity are still not very distinguishable for me and if I use $black-60-opacity, it is very dark.

For now I am have used $black-12-opacity and $black-20-opacity colours.

Suggestion: we can define background colour for selected option and hover and have it consistent everywhere in website. Also we can have some wide range of colours.

Reply:
Yes, we have to go through the color palettes and clean it up anyway. This will be all part of the redesign task: DSP-664.

The main problem will be the style configuration of dsp-ui from outside. We should have an implementation of theme configuration to overwrite default dsp-ui styles in the app.

Reply:
@waychal I think we can merge this PR now. And then I can do new release of UI-Lib today and also update and release the app.